server/src/test/java/org/opensearch/search/aggregations/support/IncludeExcludeTests.java

```diff
@@ -58,23 +54,14 @@
 
 public class IncludeExcludeTests extends OpenSearchTestCase {
 
-    private final IndexSettings dummyIndexSettings = new IndexSettings(
-        IndexMetadata.builder("index")
-            .settings(Settings.builder().put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT))
-            .numberOfShards(1)
-            .numberOfReplicas(0)
-            .build(),
-        Settings.EMPTY
-    );
-
     public void testEmptyTermsWithOrds() throws IOException {
         IncludeExclude inexcl = new IncludeExclude(new TreeSet<>(Collections.singleton(new BytesRef("foo"))), null);
-        OrdinalsFilter filter = inexcl.convertToOrdinalsFilter(DocValueFormat.RAW, dummyIndexSettings);
+        OrdinalsFilter filter = inexcl.convertToOrdinalsFilter(DocValueFormat.RAW);
```

Comment:
Is there a test where we are overriding the limit?

Reply:
It's tested via the [Aggregations IT](https://github.com/kartg/OpenSearch/blob/2.0/server/src/internalClusterTest/java/org/opensearch/search/aggregations/AggregationsIntegrationIT.java#L101-L125)